src/Features/VisualBasic/Portable/ConvertToInterpolatedString/VisualBasicConvertRegularStringToInterpolatedStringRefactoringProvider.vb

```diff
@@ -0,0 +1,24 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Composition
+Imports System.Diagnostics.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.ConvertToInterpolatedString
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.ConvertToInterpolatedString
+    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeRefactoringProviderNames.ConvertToInterpolatedString), [Shared]>
+    Friend NotInheritable Class VisualBasicConvertRegularStringToInterpolatedStringRefactoringProvider
+        Inherits AbstractConvertRegularStringToInterpolatedStringRefactoringProvider
+
+        <ImportingConstructor>
+        <SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification:="Used in test code: https://github.com/dotnet/roslyn/issues/42814")>
+        Public Sub New()
+        End Sub
+
+        Protected Overrides Function SupportsConstantInterpolatedStrings(document As Document) As Boolean
+            Return False
+        End Function
```

Comment:
do we already have a test for this case in VB?

Reply:
I don't know VB, but by manual testing I verified, that there is no support for constant interpolated strings in the lates version of VB. However, I didn't write any explicit unit test for situation like "in this scenario in C# we would provide a refactoring but in VB we must not", because it is not interesting to test "Return False" IMO. Should I write such test?)

Reply:
it's definitely worthwhile to add such a test :)